src/DynamoRevit/DynamoRevit.cs

```diff
@@ -253,7 +253,13 @@ public Result ExecuteCommand(DynamoRevitCommandData commandData)
                 if (CheckJournalForUiDisplay(extCommandData))
                 {
                     dynamoViewModel = InitializeCoreViewModel(revitDynamoModel);
+
+                    // Let the host (e.g. Revit) control the rendering mode
+                    var save = RenderOptions.ProcessRenderMode;
                     InitializeCoreView().Show();
+                    RenderOptions.ProcessRenderMode = save;
+                    revitDynamoModel.Logger.Log("WPF Render Mode: " + RenderOptions.ProcessRenderMode.ToString());
```

Comment:
I know it might not be required, but do you think we should put "WPF Render Mode" into resource string so that they could be localized? Cause the render mode messages seem to be important to user to know.